src/three-components/Renderer.ts

```diff
@@ -96,6 +102,12 @@ export class Renderer extends EventDispatcher {
       this.renderer.physicallyCorrectLights = true;
       this.renderer.setPixelRatio(resolveDpr());
 
+      if (options != null && options.debug) {
+        this.debugger = new Debugger(this);
+      } else {
+        this.renderer.debug = {checkShaderErrors: false};
```

Comment:
if this condition is going to live here, maybe turning it on should live here too instead of buried in the Debugger.ts?

Reply:
Yeah, it's funky because it's technically on by default, I just wanted the `Debugger` to assert that it be on when it is used.

Reply:
I changed the logic here to make it more explicit. LMKWYT.